src/content/glossary/index.md

```diff
@@ -530,6 +530,10 @@ A network [node](#node) that finds valid [proof-of-work](#pow) for new blocks, b
   Mining
 </DocLink>
 
+### Mint {#mint}
+
+It refers to the process of issuing new coins/tokens.
```

Comment:
I think we need a better, more detailed description here. 

Issuing isn't the right word here IMO. Minting is explicitly **creating** new currency that didn't previously exist.